pom.xml

```
@@ -384,30 +384,28 @@
 
 
         <!-- ###  PAYARA  ### -->
-
         <profile>
-            <id>payara4</id>
+            <id>Payara Community</id>
             <activation>
                 <property>
                     <name>payara.version</name>
-                    <value>/4.*/</value>
+                    <value>/^[5-9]\.[0-9]{4}\.[0-9]{1,2}.*/</value>
```

```suggestion
                    <value>/^[5-9]\.[0-9]{4}\.[0-9].*/</value>
```
Just spotted that bit isn't necessary